### PR TITLE
Added database function for incrementing Webopoly wins

### PR DESCRIPTION
**Pull request by nathan-weinberg**

### CODE REVIEW

File: database.py
Comment (timflannagan) [inline database.py]: is there something wrong with this indentation?

**Comment by Daniel-Pivonka** [comment]
lets wait for @adk3798 to review before merging

**Review by adk3798 — APPROVED** [review]
LGTM 👍

**Pull request merged**
